doc/input-output-reference/src/standard-output-reports/output-table-summaryreports.tex

```diff
@@ -172,6 +172,27 @@ \subsubsection{Predefined Annual Summary Reports}\label{predefined-annual-summar
   Minimum Outside Air During Occupied Hours table shows for each zone the average and nominal number of occupants, the zone volume, the minimum air change rate based on mechanical ventilation, infiltration and simple ventilation during occupied hours.
 \end{itemize}
 
+\paragraph{Outdoor Air Details}\label{outdoor-air-details}
+
+The Outdoor Air Details report (key: OutdoorAirDetails) produces a report that includes the following tables:
```

Comment:
OK, so this is the section that describes the new tabular report, if the user enters the `OutdoorAirDetails` key.  I was expecting to find a list of all the possible keys in this section which would need this key added, but I guess that's not here.  Just in the IDD itself.